packages/account-web-vault/README.md

```diff
@@ -1,2 +1,7 @@
+# Add Custom verida button
 
-@todo
\ No newline at end of file
+Here you can add our custom verida image url in your application so as to main our brand across application see example below.
+
+```
+<img src="https://assets.verida.io/connect-button.png" alt="Verida Vault App" />
```

Comment:
@cmcWebCode40 @nick-verida 

`https://assets.verida.io/connect-button.png` does not exist